src/components/chart/_macro-options.md

```diff
@@ -1,6 +1,6 @@
 | Name                    | Type    | Required | Description                                                                                                                                                                                  |
 | ----------------------- | ------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| chartType               | string  | true     | The type of chart to render. Supported types include: 'line', 'bar' and 'column'.                                                                                                            |
+| chartType               | string  | true     | The type of chart to render. Supported types include: 'line', 'bar', 'column', 'area' and 'scatter'.                                                                                         |
```

Comment:
```suggestion
| chartType               | string  | true     | The type of chart to render. Supported types include: 'line', 'bar', 'column', and 'area'.                                                                                                   |
```
I know its small but to keep changes separate, I think we should only be adding in changes that relate to this PR

Reply:
Now updated.